problem-solver/cxx/exampleModule/keynodes/Keynodes.cpp

```diff
@@ -0,0 +1,9 @@
+
+#include "Keynodes.hpp"
+
+namespace exampleModule
+{
+    ScAddr Keynodes::action_letter_search;
+    ScAddr Keynodes::concept_message_about_letter_search;
+    ScAddr Keynodes::rrel_entity;
```

Comment:
The rrel_entity keynode is used in 2 modules. Move rrel_entity to commonKeynodes and use it here and in Interface agent.